spec/paperclip/has_attached_file_spec.rb

```diff
@@ -38,6 +38,14 @@
       assert_adding_attachment('avatar').defines_callback('after_commit')
     end
 
+    it 'defines an after_commit callback if the class allows after_commit callbacks' do
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.